### PR TITLE
Add changelog with 3.0.0rc3

### PR DESCRIPTION
**Pull request by BIGWangYuDong**

### CODE REVIEW

**Comment by BIGWangYuDong** [comment]
https://github.com/open-mmlab/mmdetection/pull/9058 should be merged first

File: docker/Dockerfile [hunk 1]
```diff
@@ -29,7 +29,7 @@ RUN apt-get update \
 
 # Install MMEngine and MMCV
 RUN pip install openmim && \
-    mim install "mmengine==0.2.0" "mmcv>=2.0.0rc1"
+    mim install "mmengine==0.3.0" "mmcv>=2.0.0rc2"
```
Comment (hhaAndroid) [inline docker/Dockerfile:32]: The mmcv version should not need to be changed

File: docs/en/notes/changelog.md [hunk 2]
```diff
@@ -1,5 +1,48 @@
...
+## v3.0.0rc2 (4/11/2022)
+
+### Highlights
+
+- Support training detection models in Detectron2
+
+#### New Features
+
+- Support training detection models in Detectron2 (#8672)
+- Support [CrowdDet](https://arxiv.org/abs/2003.09163) (#8744)
```
Comment (hhaAndroid) [inline docs/en/notes/changelog.md:12]: This can also be moved to Highlights

File: docs/en/notes/changelog.md [hunk 3]
```diff
@@ -1,5 +1,48 @@
...
+- Support [EIoU Loss](https://ieeexplore.ieee.org/document/9429909) (#9086)
+
+#### Bug Fixes
+
+- Fix `XMLDataset` image size error (#9216)
+- Fix buugs of empty_instances when predicting without nms in roi_head (#9015)
```
Comment (RangiLyu) [inline docs/en/notes/changelog.md:18]:
```suggestion
- Fix bugs of empty_instances when predicting without nms in roi_head (#9015)
```

**Review by RangiLyu — COMMENTED** [review]
Also, update README.md

File: docs/en/notes/changelog.md [hunk 4]
```diff
@@ -1,5 +1,49 @@
...
+
+- Support training detection models in Detectron2 (#8672)
+- Support [CrowdDet](https://arxiv.org/abs/2003.09163) (#8744)
+- Support [EIoU Loss](https://ieeexplore.ieee.org/document/9429909) (#9086)
+
+#### Bug Fixes
+
+- Fix `XMLDataset` image size error (#9216)
+- Fix bugs of empty_instances when predicting without nms in roi_head (#9015)
+- Fix detr_r18_config (#9158)
```
Comment (ZwwWayne) [inline docs/en/notes/changelog.md:20]: use complete sentence

File: docs/en/notes/changelog.md [hunk 5]
```diff
@@ -1,5 +1,50 @@
 # Changelog of v3.x
 
+## v3.0.0rc2 (4/11/2022)
+
```
Comment (ZwwWayne) [inline docs/en/notes/changelog.md:4]: Add a line to indicate that we upgrade the version of MMEngine due to #9058, or we can update line 24.

File: docs/en/notes/changelog.md [hunk 6]
```diff
@@ -1,5 +1,50 @@
...
+### Highlights
+
+- Support training detection models in Detectron2
```
Comment (ZwwWayne) [inline docs/en/notes/changelog.md:7]: PR ID

**Pull request merged**